addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -51,11 +51,7 @@
 </template>
 
 <template id="s_newsletter_subscribe_popup" name="Newsletter Popup">
-    <div class="o_newsletter_popup" data-list-id="0">
-        <div class="alert alert-warning css_non_editable_mode_hidden o_not_editable" role="alert">
-            <strong>Newsletter Popup!</strong> The newsletter popup is active on this page. <a class="o_edit_popup" data-toggle="modal" href="#">Edit Popup</a>
-        </div>
-    </div>
+    <div class="o_newsletter_popup o_snippet_invisible" data-list-id="0" string="A newsletter popup is active on this page."/>
```

Comment:
Mmh kinda annoying to be forced to produce invalid DOM (string attribute) for the sentence to be translated... maybe you can find a better solution?

Reply:
Indeed, used `title` as discussed